### PR TITLE
Add compat data for HTMLHeadingElement

### PR DESCRIPTION
**Pull request by gsouquet**
https://developer.mozilla.org/en-US/docs/Web/API/HTMLHeadingElement

### CODE REVIEW

File: api/HTMLHeadingElement.json [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+            "version_added": true
+          },
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": true
+          },
+          "firefox": {
+            "version_added": true
```
Comment (ExE-Boss) [inline api/HTMLHeadingElement.json]: [Should be `1`.](https://developer.mozilla.org/en-US/docs/Web/API/HTMLHeadingElement#Browser_compatibility)

File: api/HTMLHeadingElement.json [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+          },
+          "firefox_android": {
+            "version_added": true
```
Comment (ExE-Boss) [inline api/HTMLHeadingElement.json]: [Should be `4`.](https://developer.mozilla.org/en-US/docs/Web/API/HTMLHeadingElement#Browser_compatibility)

File: api/HTMLHeadingElement.json [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+              "version_added": true
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": true
```
Comment (ExE-Boss) [inline api/HTMLHeadingElement.json]: [Should be `1`.](https://developer.mozilla.org/en-US/docs/Web/API/HTMLHeadingElement#Browser_compatibility)

File: api/HTMLHeadingElement.json [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+            },
+            "firefox_android": {
+              "version_added": true
```
Comment (ExE-Boss) [inline api/HTMLHeadingElement.json]: [Should be `4`.](https://developer.mozilla.org/en-US/docs/Web/API/HTMLHeadingElement#Browser_compatibility)

**Comment by gsouquet** [comment]
@ExE-Boss I've made a conscious decision to put `true` instead of the version number to be consistent with the other values. We could probably put any other browser first version instead of `true`. Happy to change if needed.

**Pull request merged**